doc/classes/DisplayServer.xml

```diff
@@ -1088,18 +1088,18 @@
 			<return type="Image" />
 			<param index="0" name="screen" type="int" default="-1" />
 			<description>
-				Returns screenshot of the [param screen].
+				Returns a screenshot of the [param screen].
 				[b]Note:[/b] This method is implemented on Linux (X11), macOS, and Windows.
-				[b]Note:[/b] On macOS, this method requires "Screen Recording" permission, if permission is not granted it will return desktop wallpaper color.
+				[b]Note:[/b] On macOS, this method requires the "Screen Recording" permission. If permission is not granted, this method returns a screenshot that will not include other application windows and OS elements not related to the application.
```

Comment:
```suggestion
				[b]Note:[/b] On macOS, this method requires the "Screen Recording" permission. If permission is not granted, this method returns a screenshot that will not include other application windows or OS elements not related to the application.
```
Final detail, think "I don't have a dog or a cat", in this case this means "it won't include A and B together, but can include A, or B, or neither"